python/ray/autoscaler/v2/instance_manager/config.py

```diff
@@ -388,7 +388,7 @@ def get_max_concurrent_launches(self) -> int:
 
     def disable_node_updaters(self) -> bool:
         provider_config = self._configs.get("provider", {})
-        return provider_config.get(DISABLE_NODE_UPDATERS_KEY, True)
+        return provider_config.get(DISABLE_NODE_UPDATERS_KEY, False)
```

Comment:
All providers except KubeRay should have updaters; therefore, the default of `DISABLE_NODE_UPDATERS_KEY` should be `False`. KubeRay has already set `DISABLE_NODE_UPDATERS_KEY=True` explicitly.